DEVELOPMENT.md

```diff
@@ -162,8 +162,36 @@
     )
     ```
 
-4.  Build the new target.
+4. Point to local gapic-generator-java
+   
+   Normally, googleapis's build pulls in googleapis/gapic-generator-java from the
+   Internet:
+
+   ```
+   # Java microgenerator.
+   …
+   _gapic_generator_java_version = "2.1.0"
+   
+   http_archive(
+       name = "gapic_generator_java",
+       …
+       urls = ["https://github.com/googleapis/gapic-generator-java/archive/v%s.zip" % _gapic_generator_java_version],
+   )
+   ```
+
+   By replacing this portion using the built-in local_repository rule, you can mak
+   it refer to your local development repo:
+
+   ```
+   local_repository(
+     name = "gapic_generator_java",
+     path = "/home/<your id>/gapic-generator-java",
+   )
+   ```
+
+5. Build the new target.
```

Comment:
Steps looks good; wondering if you wanted to add a step on how to build any client library (e.g. technically you don't need to build the showcase service, can build any service). Something like:

You can generate any client library based on the protos within `googleapis`. You just need the name of the service within the `java_gapic_assembly_gradle_pkg` rules within the services `BUILD.bazel` file. For instance, if you wanted to test your local generator on the `speech` service, you can run:

`bazel build google-cloud-speech-v1-java`

Reply:
Good. I added 83d2607